pkg/netset/connectionset_test.go

```diff
@@ -122,6 +139,20 @@ func TestConnectionSetBasicOperations(t *testing.T) {
 	// conns: all
 	fmt.Printf("conn9UnionConn6 cubes string:\n%s\n", getPartitionsStr(conn9UnionConn6))
 
+	//conn12 cubes string:
+	// src: 10.240.10.0/24,
+	// dst: 10.240.10.1/32, 10.240.10.4/30, 10.240.10.8/29, 10.240.10.16/28, 10.240.10.32/27, 10.240.10.64/26, 10.240.10.128/25,
+	// conns: protocols 0, src-ports 1-65535, dst-ports 1-65535;
+	// src: 10.240.10.0/25,
+	// dst: 10.240.10.2/31,
+	// conns: protocols 1, src-ports 1-65535, dst-ports 53,protocols 0, src-ports 1-65535, dst-ports 1-65535;all icmp
+	// src: 10.240.10.128/25,
+	// dst: 10.240.10.2/31,
+	// conns: protocols 0, src-ports 1-65535, dst-ports 1-65535;all icmp
+	fmt.Printf("conn12 cubes string:\n%s\n", getPartitionsStr(conn12))
+
+	fmt.Printf("conn17 cubes string:\n%s\n", getPartitionsStr(conn17))
+
 	fmt.Println("done")
```

Comment:
This is not a test, it's only printing. Should I transform it into a test?

Reply:
The relevant `require` checks are already in the test, added this printing for better visibility and documentation of the objects created in the test.